beep/structure.py

```diff
@@ -132,29 +132,37 @@ def from_file(cls, path, validate=False):
         else:
             raise ValueError("{} does not match any known file pattern".format(path))
 
-    def get_interpolated_discharge_cycles(self, v_range=None, resolution=1000):
+    def interpolated_cycles(self, v_range, resolution, step_type='discharge'):
```

Comment:
Is there a more descriptive name for this method that has an active verb and distinguishes it more from `get_interpolated_cycles`?  Maybe `get_interpolated_steps`?

Reply:
Yeah, `interpolated_cycles` is also similar to `cycles_interpolated` attribute of the `processed_cycler_run` object.  I am fine with 'get_interpolated_steps` or `interpolate_within_step_type`

Reply:
Agreed, I think the methods should begin with `get` I will change that